### DISCUSSION TITLE
Can I decalre a type of (`a -> `a)

### DISCUSSION THREAD

**Discussion by Xyncgas** [fsharp/fslang-suggestions#1168] [General]
I tried doing
``` F#
type Control<`a> = `a -> `a
```

**Comment by jl0pd** [comment]
Such kind of questions are better suited for StackOverflow.

Answering question: you can do that, just use quote  instead of backtick:
```fsharp
type Control<'a> = 'a -> 'a
```

Backtick is used when you want to use identifiers with space or other characters, e.g.

```fsharp
let ``weird operator!`` x = x
printfn "%d" (``weird operator!`` 1)
```

Quote is used when working with generic types https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/generics/
